Index: app/src/main/java/devandroid/gilberto/applistacurso/controller/PessoaController.java
===================================================================
diff --git a/app/src/main/java/devandroid/gilberto/applistacurso/controller/PessoaController.java b/app/src/main/java/devandroid/gilberto/applistacurso/controller/PessoaController.java
--- a/app/src/main/java/devandroid/gilberto/applistacurso/controller/PessoaController.java	
+++ b/app/src/main/java/devandroid/gilberto/applistacurso/controller/PessoaController.java	
@@ -1,4 +1,11 @@
 package devandroid.gilberto.applistacurso.controller;
 
+import devandroid.gilberto.applistacurso.model.Pessoa;
+
 public class PessoaController {
+
+
+    public void salvar(Pessoa pessoa) {
+
+    }
 }
